tasks/acceptance-tests.yml

```diff
@@ -0,0 +1,65 @@
+---
+
+# acceptance-tests installer
+
+- name: "Checkout out acceptance tests repository"
+  git:
+    repo: "https://github.com/artefactual-labs/archivematica-acceptance-tests.git"
+    dest: "{{ archivematica_src_dir }}/archivematica-acceptance-tests"
+    version: "{{ archivematica_src_acceptance_tests_version }}"
+    force: "yes"
+
+- name: "Install pip dependencies in virtualenv"
+  pip:
+    chdir: "{{ archivematica_src_dir }}/archivematica-acceptance-tests"
+    requirements: "requirements.txt"
+    virtualenv: "/usr/share/python/acceptance-tests"
+    virtualenv_python: "python3.4"
+    state: latest
+
+- name: "Does the Google apt file exist?"
+  command: "test -f /etc/apt/sources.list.d/google-chrome.list"
+  register: google_apt_exists
+  ignore_errors: True
```

Comment:
I think that the values in `archivematica_src_acceptance_tests_browser_list` should control whether Chrome itself is installed, not just whether chromedriver is installed. Is there a reason that you did not do it this way?